gobblin-utility/src/main/java/gobblin/util/HadoopUtils.java

```diff
@@ -110,6 +113,7 @@ public static Configuration newConfiguration() {
     // Add a new custom filesystem mapping
     conf.set("fs.sftp.impl", "gobblin.source.extractor.extract.sftp.SftpLightWeightFileSystem");
     conf.set("fs.sftp.impl.disable.cache", "true");
+    conf.set("fs.adl.impl", "org.apache.hadoop.fs.adl.AdlFileSystem");
```

Comment:
I see, if you add an ADL module you could instead use the service provider pattern with your own resource. Have you considered submitting a PR to their project fixing this?